scripts/domManipulation.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable comma-dangle */
+/* eslint-disable prefer-const */
+const booksLocalStorage = JSON.parse(localStorage.getItem('books') || '[]');
+let books = booksLocalStorage;
+const booksContainer = document.querySelector('.books-container');
+
+function displayBooks() {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.